tsconfig.json

```diff
@@ -2,6 +2,7 @@
 	"extends": "@sindresorhus/tsconfig",
 	"compilerOptions": {
 		"noEmit": true,
+		"noUnusedLocals": false, // Allow unused variables in test-d/*.ts files
```

Comment:
I had the same concern before. but `.d.ts` seem not be affected after my test, I guess that's because type is not local variables but I don't know real reason now.